async-nats/src/lib.rs

```diff
@@ -451,6 +452,8 @@ impl ConnectionHandler {
         }
 
         impl<'a> ProcessFut<'a> {
+            const RECV_CHUNK_SIZE: usize = 16;
```

Comment:
What is the reason for picking 16?
Were you benching other values?

Reply:
I've tried other values but the results are mixed. It doesn't hurt too much to increase this number, but it's basically like forcing a higher `ConnectOptions::client_capacity` value.